lib/open_api_spex/plug/cast_and_validate.ex

```diff
@@ -49,6 +49,30 @@ defmodule OpenApiSpex.Plug.CastAndValidate do
           |> Conn.send_resp(400, msg)
         end
       end
+
```

Comment:
This isn't how I understand the meaning of `readOnly` and `writeOnly` from https://swagger.io/docs/specification/data-models/data-types/

> readOnly properties are included in responses but not in requests, and writeOnly properties may be sent in requests but not in responses

It shouldn't matter the HTTP verb that is being used. What matters is whether the schema is being cast from a request (most of what OpenApiSpex is used for) or a response - typically only used in tests or if you are using OpenApiSpex in conjunction with an HTTP client to make external requests.

Am I misunderstanding?